docs/proposals/31-inplace-node-update.md

```diff
@@ -101,34 +113,38 @@ The only prerequisite is that the machine image/OS on the node must support in-p
-3. `machine-controller-manager` will select the nodes based on the configured `maxUnavailable` and label them with `selected-for-update`. This label can be used to identify nodes that are currently undergoing an update.
-4. `machine-controller-manager` will cordon and drain the node and label it with `ready-for-update` label.
-5. `gardener-node-agent` will detect the `ready-for-update` label and perform the update on the node. And it will wait for the update to complete within the specified timeout.
-6. Once the Kubernetes/Machine image version is updated, the node will be labeled by the `gardener-node-agent` with `update-successful` label.
-7. `machine-controller-manager` will `uncordon` the node, and remove all the added labels. The node is then ready to schedule the workloads.
+#### `ManualInPlaceUpdate` strategy
 
-![InplaceUpdate](assets/inplace-update.png "InPlace Update Strategy")
+1. On a shoot spec change for a worker pool, `machine-controller-manager` will label all nodes of the changed worker pool with the `node.machine.sapcloud.io/candidate-for-update` label. This label is used to identify nodes that require an update.
+2. `machine-controller-manager` will add the necessary annotation to the nodes to prevent them from being scaled down by `cluster-autoscaler` during the update process.
+3. `machine-controller-manager` will wait for the `node.machine.sapcloud.io/selected-for-update` label on the node. The orchestration of the update is solely the responsibility of the user. The user is free to select the nodes to be updated at will.
+4. Once the `node.machine.sapcloud.io/selected-for-update` label is added by the user, steps 4-7 of the `AutoInPlaceUpdate` strategy will be performed.
 
-#### InplaceUpdateOnLabel strategy
+![ManualInPlaceUpdate](assets/31-manual-inplace-update.png "Manual In Place Update Strategy")
 
-1. On shoot spec change, nodes will be labeled with `candidate-for-update`.
-2. `machine-controller-manager` will add the necessary annotation to the nodes to prevent it from being scaled down by `cluster-autoscaler` during the update process.
-3. `machine-controller-manager` will wait for the `selected-for-update` label on the node. The orchestration of the update is solely the responsibility of the user. They are free to select the nodes to be updated at will.
-4. Once the `selected-for-update` label is added by the user on the node, steps 4-7 of the `Inplace` strategy will performed.
+A new optional [constraint](https://github.com/gardener/gardener/blob/master/docs/usage/shoot/shoot_status.md#constraints) will be introduced in the Shoot status to indicate that some worker pools are undergoing a manual in-place update. While the Shoot will be marked as successfully reconciled, these worker pools may still be outdated. A new field will be added to the Shoot status to keep track of the pending worker pools.
```

Comment:
> A new field will be added to the Shoot status to keep track of the pending worker pools.

Can you make this more specific and show an example how this looks like? Perhaps you also want to link to #11027 which introduces the API you plan to augment?